addons/survey/models/survey_survey.py

```diff
@@ -276,36 +278,30 @@ def _compute_has_conditional_questions(self):
         for survey in self:
             survey.has_conditional_questions = any(question.is_conditional for question in survey.question_and_page_ids)
 
-    @api.onchange('scoring_success_min')
-    def _onchange_scoring_success_min(self):
-        if self.scoring_success_min < 0 or self.scoring_success_min > 100:
-            self.scoring_success_min = 80.0
-
-    @api.onchange('scoring_type')
-    def _onchange_scoring_type(self):
-        if self.scoring_type == 'no_scoring':
-            self.certification = False
-            self.is_time_limited = False
+    @api.depends('scoring_type')
+    def _compute_certification(self):
+        for survey in self:
+            if survey.certification is None or survey.scoring_type == 'no_scoring':
```

Comment:
Depends only on scoring_type, so why the "is None" ?

Reply:
I'm not a big fan of those None either, but I'm guessing problems related to them are not solved yet because:
![image](https://user-images.githubusercontent.com/41282536/78867431-61c53b00-7a41-11ea-9815-9e5ba9375165.png)

What should I do then? :(